docs/extending/extending_subitems_list.md

```diff
@@ -94,4 +94,12 @@ Optionally, Sub-items module can take a following list of props:
     - **tableView** _{Object}_ - list of table view component labels
     - **tableViewItem** _{Object}_ - list of table item view component labels
     - **loadMore** _{Object}_ - list of load more component labels
-    - **gridViewItem** _{Object}_ - list of grid item view component labels
\ No newline at end of file
+    - **gridViewItem** _{Object}_ - list of grid item view component labels
+
+## Reusing Sub-items list
+
+To add a Sub-items list on a page that does not have the (right) action sidebar, you need to do one of the following things:
+
+- add a `<div>` element with the `.ez-extra-actions-container` selector
+- change the selector in the Sub-items settings by sending the `languageContainerSelector` prop
```

Comment:
I think this `languageContainerSelector` should be also added to the option props above.